sql_upgrade.php

```diff
@@ -86,14 +97,19 @@
   require("acl_upgrade.php");
```

Comment:
oops, I forgot about the acl_upgrade.php script that is called above which does stuff with the acl version...